basic-algorithm-scripting/find-the-longest-word-in-a-string.js

```diff
@@ -0,0 +1,13 @@
+function findLongestWordLength(str) {
+  let longestWord = str.split(" ");
+  let longestWordLength = 0;
+  
+  for (let i = 0; i < longestWord.length; i++) {
```

Comment:
change this loop for based on index to a loop of for using word as the name of control loop variable.

Reply:
mude a forma do loop de ao inves de fazer o loop pelo index fazer o loop por elemento do vetor.
Utilize para isso o for... of e use o nome word  para nomear a variavel controle do loop